packages/dlp/src/index.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2016 Google Inc. All rights reserved.
```

Comment:
Will be updated in the generator with https://github.com/googleapis/toolkit/pull/1293